src/Tokenizer.php

```diff
@@ -898,7 +898,7 @@ private function createNextToken(string $string, Token|null $previous = null): T
             ) {
                 return new Token(
                     Token::TOKEN_TYPE_RESERVED_TOPLEVEL,
-                    substr($upper, 0, strlen($matches[1])),
+                    substr($string, 0, strlen($matches[1])),
```

Comment:
There is something with the coverage, I belive this is tested by phpunit, please help.